### ISSUE TITLE
microsoft teams shows black window with red border

### ISSUE THREAD

**Issue by codygman** [NixOS/nixpkgs#95202]
**Describe the bug**
When trying to screenshare in exwm with teams 1.3.00.16851 you'll just get a red border and black screen under kde/exwm.

Workaround: screenshare in kde before starting exwm

possibly related #87069

I guess that means within exwm the screenshare window isn't composited? not sure.

The recommended workaround on teams is to rename `rect-overlay` so it won't be used, but I'm not sure it would have helped here. If it does fix the issue, maybe the nix package should do that renaming workaround. See the accepted answer [in this microsoft teams bug](https://docs.microsoft.com/en-us/answers/questions/42095/sharing-screen-not-working-anymore-bug.html).

**To Reproduce**
1. be in kde and start emacs, replace window manager with exwm
2. call someone and screenshare on teams
3. see black screen with red border

**Expected behavior**

What was a black screen is now transparent and screen sharing works as expected.

**Screenshots**

**Additional context**

**Notify maintainers**

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

 - system: `"x86_64-linux"`
 - host os: `Linux 5.7.2, NixOS, 20.09pre229574.0a146054bdf (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.6`
 - channels(root): `"nixos-20.09pre229574.0a146054bdf"`
 - channels(cody): `"home-manager"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by jonringer** [comment]
under x11, I get normal visuals. Audio was bad, but haven't tried it recently

**Comment by codygman** [comment]
I fixed this by not launching exwm under kde, and instead launching exwm with xsession and adding the compton (or whatever it was renamed to) service for compositing. That means the kde compositor doesn't work with exwm I guess?

I'm not sure if this should be closed, a bug should be opened for exwm's package for not handling this, or if this is something that nix shouldn't handle.

**Comment by liff** [comment]
This looks like duplicate of #99279. @codygman does the latest version work for you now that the fix is in 20.09 and unstable?

**Comment by jonringer** [comment]
I believe this was fixed
